gemfiles/Gemfile.rails-6.1-stable

```diff
@@ -5,3 +5,4 @@ source "https://rubygems.org"
 gemspec path: ".."
 
 gem "activerecord", github: "rails/rails", branch: "6-1-stable"
+gem "sqlite3", "~> 1.4"
```

Comment:
The newer version (2.x.x) is released but the older rails versions are still pin to use a lower version (~> 1.4).
[Reference PR](https://github.com/rails/rails/pull/51592)
Only the latest version (7.2.x) is able to use a higher version.